src/main/resources/cloud-insight-for-jira.properties

```diff
@@ -38,17 +38,20 @@ ci.atlassianplugin.boostrap.msg.connectionerror =An error occurred during connec
 ci.constant.category =CI
 ci.constant.custom.remediationItem =Remediation Item
 ci.constant.custom.remediationId =Remediation ID
+ci.constant.custom.incidentId =Incident ID
 ci.constant.custom.groupAssigned =Group Assigned
 
-ci.constant.screen.name = Cloud Insight Custom Screen
-ci.constant.scheme.name = Cloud Insight Integration
-ci.constant.scheme.description = Cloud Insight Integration
-ci.constant.issuetypescreen.name = Cloud Insight Screen Scheme
+ci.constant.screen.name =Alert Logic Custom Screen
+ci.constant.scheme.name =Alert Logic Integration
+ci.constant.scheme.description = Alert Logic Integration
+ci.constant.issuetypescreen.name = Alert Logic Screen Scheme
 ci.constant.issuetypescreen.description = Cloud Insight Issue Type
-ci.constant.issuetype.description =A Cloud Insight remediation step that contains information about the exposure to be addressed. If you properly disposed the remediation, the remediation step does not appear after the next scan.
+ci.constant.issuetype.description =A remediation step that contains information about the exposure to be addressed. If you properly disposed the remediation, the remediation step does not appear after the next scan.
 ci.constant.issuetype.name =Cloud Insight Remediation
-ci.constant.issuetypeschema.name =Cloud Insight Remediation Issue Type Scheme
-ci.constant.issuetypeschema.description =CI Remediation Issue Type Scheme
+ci.constant.issuetype.incident.name =Alert logic Incident
+ci.constant.issuetype.incident.description =An Alert logic incident type
```

Comment:
Alert logic should be "Alert Logic", both caps

Reply:
thanks Jason, done